README.md

```diff
@@ -162,3 +164,24 @@ void TRC_register_err_log_callback(TRC_err_log_callback callback);
 Sending NULL to this function will re-register the default tracy log handler.
 
 
+### Allocating the trace stack
```

Comment:
This is necessary for the program to link, so I think it should be more to the top in the README.
I think that between the `example` and the `program structure` part is fine.